### PR TITLE
Update to maintenance checklist

### PR DESCRIPTION
**Pull request by ChrisMcGowan**
## Changes proposed in this pull request:
- From the quarterly account review - updated steps used for platform maintenance checklist

## Security Considerations
Keeping updated documentation is part of our SSP

### CODE REVIEW

File: _docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -91,57 +91,19 @@ You can also view this information in each of our four environments, `tooling`,
...
+The Concourse jobs for deploying `production-cf` and `production-logsearch` are setup for manual operator release.  Keep track of when changes are ready to deploy in production since these are customer facing and make sure deployment of these jobs are done during normal platform support hours and during pre-determined times as described [here](https://cloud.gov/docs/overview/customer-service-objectives/). 
 
-> [Get familiar with the documentation for CloudTrail logs](http://docs.aws.amazon.com/IAM/latest/UserGuide/cloudtrail-integration.html).
+## Monitor AWS alerts
 
-Use the [AWS Console](http://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-console.html)
-to [review API activity history](http://docs.aws.amazon.com/awscloudtrail/latest/userguide/view-cloudtrail-events-console.html)
-for the _EventNames_ listed below.
-Or, use the AWS CLI with the appropriate `$event_name`, and parse the emitted JSON:
+Keep tabs on the private channel `#cg-aws-security` and review the alerts that are posted by AWS GuardDuty.  Review the linked CloudWatch alert for more details and research what triggered the alert.
```
Comment (markdboyd) [inline _docs/ops/maintenance-list.md]: Alerts from GuardDuty go to the Google Group, not the Slack channel

People should also monitor `#cg-aws-security` for the alerts on failed logins, etc

**Comment by ChrisMcGowan** [comment]
Pages preview of the change: https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.sites.pages.cloud.gov/preview/cloud-gov/cg-site/mainlist-update/docs/ops/maintenance-list/

**Comment by ChrisMcGowan** [comment]
@markdboyd  can you 👀 this as you where part of this meeting - Thanks

File: _docs/ops/maintenance-list.md (hunk 1 above)
Comment (ChrisMcGowan) [reply _docs/ops/maintenance-list.md]: ok - misunderstanding - i'll add an additional sub heading for this.

Comment (ChrisMcGowan) [reply _docs/ops/maintenance-list.md]: @markdboyd changes made in new commit

File: _docs/ops/maintenance-list.md [hunk 2]
```diff
@@ -91,57 +91,23 @@ You can also view this information in each of our four environments, `tooling`,
 
 The platform's CI builds can be automatically triggered by third-party resources when updates are released (ie. source code, stemcell). Monitor the `#cg-platform` and `#cg-platform-news` Slack channels for notifications about any failed builds. For any automated builds that failed, review the logs of the failed build task in Concourse, and escalate the failure to platform ops if a cursory triage and rebuild does not fix this issue.
 
-## Review AWS CloudTrail events
+### Monitor cg-provision
```
Comment (markdboyd) [inline _docs/ops/maintenance-list.md]:
```suggestion
### Monitor terraform-provision pipeline
```

File: _docs/ops/maintenance-list.md [hunk 3]
```diff
@@ -91,57 +91,23 @@ You can also view this information in each of our four environments, `tooling`,
...
 
-### Easy way
+As part of platform automation, and Concourse and Terraform run a `plan-bootstrap-<environment>` job if parts of our infrastructure have changed due to PR or automation. Alerts for reviewing Terraform plan runs are located in `#cg-platoform`.  Review the plan updates and coordinate a time to run `bootstrap-<environment>` so the changes to the infrastructure can be made. All `bootstrap-<environment>` Concourse jobs are set for manual operator release so make sure the review of `plan-bootstrap-<environment>` and running `bootstrap-<environment>` are done in a timely fashion.  This is needed to keep parts of the platform current like platform certificates as example.
```
Comment (markdboyd) [inline _docs/ops/maintenance-list.md]:
```suggestion
As part of platform automation, and Concourse and Terraform run a `plan-bootstrap-<environment>` job if parts of our infrastructure have changed due to PR or automation. Alerts for reviewing Terraform plan runs are located in `#cg-platform`.  Review the plan updates and coordinate a time to run `bootstrap-<environment>` so the changes to the infrastructure can be made. All `bootstrap-<environment>` Concourse jobs are set for manual operator release so make sure the review of `plan-bootstrap-<environment>` and running `bootstrap-<environment>` are done in a timely fashion.  This is needed to keep parts of the platform current like platform certificates as example.
```

**Comment by ChrisMcGowan** [comment]
@markdboyd Looks good now

**Review by rcgottlieb — APPROVED** [review]

**Pull request merged**